examples/testgputext/fonts/license.txt

```diff
@@ -0,0 +1,37 @@
+License
```

Comment:
Please don't include any fonts in the SDL_ttf project. There are plenty out there already for people to use.

Reply:
Copied from discord.
> I actually decided to ship the font because if the font isn't monospaced the example will look quite weird and also different fonts will change the size of the text drastically due to the 3D rotation, it will be very hard to mitigate that only by changing the point size.